book/src/bridge/extern_cppqt.md

```diff
@@ -29,6 +29,10 @@ A bridge module may contain zero or more `extern "C++Qt"` blocks.
 This complements the [`extern "C++"` CXX section](https://cxx.rs/extern-c++.html)
 but allows for declaring Qt specific features on C++ types.
 
+## Automatic case conversion
+
+Automatically converting to camel or snake case can be done through an attribute for extern "RustQt" and "C++Qt" blocks, see [here](./attributes.md#automatic-case-conversion)
```

Comment:
Would change this to the following and remove the header, so it's just a sentence in the previous block

```
Automatically converting to camel or snake case can be enabled through an [attribute](./attributes.md#automatic-case-conversion).
```